internal/cmd/kafka/command_cluster_update.go

```diff
@@ -96,52 +101,44 @@ func (c *clusterCommand) update(cmd *cobra.Command, args []string, prompt form.P
 	return c.outputKafkaClusterDescription(cmd, &updatedCluster, true)
 }
 
-func (c *clusterCommand) validateResize(cmd *cobra.Command, currentCluster *cmkv2.CmkV2Cluster, prompt form.Prompt) (int32, error) {
-	// returning -1 when error or unchanged
-	if cmd.Flags().Changed("cku") {
-		cku, err := cmd.Flags().GetInt("cku")
-		if err != nil {
-			return -1, err
-		}
-		// Ensure the cluster is a Dedicated Cluster
-		if currentCluster.GetSpec().Config.CmkV2Dedicated == nil {
-			return -1, errors.New(errors.ClusterResizeNotSupportedErrorMsg)
-		}
-		// Durability Checks
-		if *currentCluster.GetSpec().Availability == highAvailability && cku <= 1 {
-			return -1, errors.New(errors.CKUMoreThanOneErrorMsg)
-		}
-		if cku <= 0 {
-			return -1, errors.New(errors.CKUMoreThanZeroErrorMsg)
+func (c *clusterCommand) validateResize(cmd *cobra.Command, cku int32, currentCluster *cmkv2.CmkV2Cluster, prompt form.Prompt) (int32, error) {
+	// Ensure the cluster is a Dedicated Cluster
+	if currentCluster.GetSpec().Config.CmkV2Dedicated == nil {
+		return -1, errors.New(errors.ClusterResizeNotSupportedErrorMsg)
+	}
+	// Durability Checks
+	if *currentCluster.GetSpec().Availability == highAvailability && cku <= 1 {
+		return -1, errors.New(errors.CKUMoreThanOneErrorMsg)
+	}
+	if cku <= 0 {
```

Comment:
If we make this a `uint`, we can simplify this
```suggestion
	if cku == 0 {
```